src/AltinnCore/Common/Helpers/CompileHelper.cs

```diff
@@ -12,10 +12,10 @@ namespace AltinnCore.Common.Helpers
     public static class CompileHelper
     {
         /// <summary>
-        /// Creates an asynchronous task for compiling an app
+        /// Creates an asynchronous task for compiling an app.
         /// </summary>
         /// <param name="compilation">The ICompilation implementation</param>
-        /// <param name="identifier">The service identifier</param>
+        /// <param name="identifier">The <see cref="ServiceIdentifier"/></param>
```

Comment:
I would suggest to undo the change. cref is manly used in summary of a class constructor, not parameter descriptions

Reply:
Allright